src/app/plugins/give.plugin.ts

```diff
@@ -0,0 +1,82 @@
+import { EmbedAuthorData, MessageEmbed } from 'discord.js';
+import Constants from '../../common/constants';
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType } from '../../common/types';
+import { PointsDocument } from '../../schemas/points.schema';
+
+export default class GivePlugin extends Plugin {
+  public commandName: string = 'give';
+  public name: string = 'Give Plugin';
+  public description: string = 'Give your tacos to someone else';
+  public usage: string = 'give @Tanndlin all\n give @Tanndlin 100';
+  public override pluginAlias = [];
+  public permission: ChannelType = ChannelType.Public;
+  public override pluginChannelName: string = Constants.Channels.Public.Games;
+
+  public override commandPattern: RegExp = /(all|\d+)/;
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public override validate(message: IMessage, args: string[]): boolean {
+    if (args.length !== 2) {
```

Comment:
Are args correct if the mention is of a multi word name? Can you test that please?

if they don’t do the right thing let’s consider stripping args of all the mentions

Reply:
It uses mentions, which in plain text is <@1111111111111> no spaces
If it doesn't follow that form its not a ping